pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -255,6 +255,11 @@ type CSIDriverSpec struct {
 	// CephFS defines CSI Driver settings for CephFS driver.
 	// +optional
 	CephFS CSICephFSSpec `json:"cephfs,omitempty"`
+	// CreateUsers determines whether CSI users and their associated secrets should be created automatically.
+	// If set to false, the user must manually manage these secrets.
+	// +kubebuilder:default=true
+	// +optional
+	CreateUsers bool `json:"createUsers,omitempty"`
```

Comment:
I don't trust the kubebuilder defaults, we have had issues with them in the past. IMO it's anyway nice to default to false. So how about this setting instead?
```suggestion
	SkipUserCreation bool `json:"skipUserCreation,omitempty"`
```